tip-247.md

```diff
@@ -0,0 +1,219 @@
+
+Notice: SR can be Verifier 
+
+
+### Comparison
+
+Basically, by adding multiple Verifiers in the current tron network shorten the block confirmation time. Current tron consensuses algorithm always require 2f+1 =19 nodes to confirm block, which take 19 slot time, while optimized tron pBFT only require 2/3+1 verifiers to confirm blocker, which take less times in comparing with former one since network request can be handled by few seconds.
+Consider about slide windows model, Current tron consensus algorithm only move one slot every pass, while optimized tron pBFT can move multiple slot once receiver 2/3+1 Verifiers replies. Below is current tron consensus algorithm:
+ ```
+     Tron window:       start..........................End
+      Block     A  ----> B->C...........17..........--->R                    (a)
+      Status  solid ---> unconfirmed->...........->unconfirmed
+ ```
+Initially, slide window starts from the first unconfirmed block, the size is 1, then will increase windows size as the number of unconfirmed block increase. Once windows size reach threshold say 19+1, which is R in above, then make B‘s status to be solid, move into C, decrease windows size by one, which is 19.And continue to move once start point block of windows become solid status.
+When combined with optimized tron pBFT, the windows can move faster.
+```
+ Tron window:     start....................................End
+ optimized:       start                  end(receives message)              (b)
+ Block     A  ----> B----->C------>D------>E..............--->R
+ Status  solid ---> unconfirmed->...................->unconfirmed
+```
+For optimized PBFT, slide window starts from the first unconfirmed block with size of 1,  will increase windows size as the number of unconfirmed block increase when 2/3+1 relies commit message from verifiers not be received.  Once get more than 2/3+1 replies commit messages from verifiers stating that block E is valid, which proves that block E on (b) is valid, which draws a conclusion that B,C,D,E are all valid and can be marked as solid status since each block hash relies precious block hash . In this case, the slide windows move super faster in comparing with tron windows before tron windows reach thresholds-19.  In best case, which means no fork and network is good means a slot time is enough to receive verifier replicas messages, both of them have the same move speed after tron windows size reach threshold. However, if there is a fork on the main chain, then tron window (a) reset windows size and wait for 19 blocks time to confirm while optimized windows (b) can maintenance the same move speed. More detail: See the proof the correctness of optimized pBFT.
```

Comment:
size of 1 -> a size of 1
maintenance - >maintain 
move speed -> movement speed